signature/src/keypair.rs

```diff
@@ -0,0 +1,46 @@
+pub trait KeyPair: core::fmt::Debug + Sized {
+    fn get_public_key_enc(&self) -> String;
+    fn get_private_key_enc(&self) -> String;
+    fn get_public_key(&self) -> ed25519_zebra::VerificationKey;
+    fn get_private_key(&self) -> ed25519_zebra::SigningKey;
+}
+
+#[derive(Debug)]
+pub struct SSIKeyPair {
```

Comment:
I think we should call this Ed25519KeyPair right? And the trait `SSIKeyPair`?

Reply:
Yep, we can do that, although it is implemented as described in the ticket: https://github.com/knox-networks/ssi/issues/39 , but i can rename, np

Reply:
If you prefer this naming convention we can keep it, I just hadn't fully realized what I had said in the issue lol